libraries/botbuilder/src/botFrameworkHttpClient.ts

```diff
@@ -53,22 +53,6 @@ export class BotFrameworkHttpClient implements BotFrameworkClient {
      * @param conversationId A conversation ID to use for the conversation with the skill.
      * @param activity Activity to forward.
      */
-    public async postActivity<T>(
```

Comment:
99.99% sure these overloads are all the same thing but maybe there's some context I'm unaware of?

Reply:
That might have been bad TypeScript on my part, but we went from untyped to typed `InvokeResponse<T>`

Reply:
I think that defaulting `T` to `any` means this should be fine. `InvokeResponse` already defaults its type param to `any` so I don't think this would break any external users.

Reply:
Sounds good, thanks!